src/main/java/com/fasterxml/jackson/annotation/JsonTypeInfo.java

```diff
@@ -119,6 +119,23 @@ public enum Id {
          */
         NAME("@type"),
 
+        /**
+         * Means that the simple name of the Java class, equivalent to the value returned by {@link Class#getSimpleName()},
+         * is used as the type identifier.
```

Comment:
as the DEFAULT type identifier (unless explicit name specified by annotation `@JsonTypeName`)